src/contract_cleaning.py

```diff
@@ -0,0 +1,19 @@
+def get_one_hot_encoded_contract(contract):
+    contract_as_list = contract.values.tolist()  # Returns a list of list [[]]
+    contract_cleaned = []
+
+    for item in contract_as_list:
+        contract_cleaned.append(item[0])
```

Comment:
I love python and stuff but this kind of code obscures what is happening imo. Checking, if it results the same I will update.